pkgs/by-name/hi/hieroglyphic/package.nix

```diff
@@ -19,18 +19,18 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "hieroglyphic";
-  version = "1.1.0";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "FineFindus";
     repo = "Hieroglyphic";
     rev = "refs/tags/v${finalAttrs.version}";
-    hash = "sha256-8UUFatJwtxqumhHd0aiPk6nKsaaF/jIIqMFxXye0X8U=";
+    hash = "sha256-bRxG+NHwHPJ2J/zTFks6M/Uy7XQ0cm8FBzlBC4yORm4=";
```

Comment:
https://github.com/NixOS/nixpkgs/pull/357401/checks?check_run_id=33280981000

Faulty upstream?

Reply:
Very weird that darwin gets a different hash. Will try to compare the directories.

Reply:
Oh, case sensitivity. I haven't encountered this kind of problem yet

Reply:
Also don't think it is...

Reply:
these files in `data/resources/icons` differ only by case:
```
c3RtYXJ5cmQtT1QxLV9sYmFn-symbolic.svg
c3RtYXJ5cmQtT1QxLV9SYmFn-symbolic.svg
```

`LV9s` -> `LV9S`